environment.yml

```diff
@@ -31,3 +31,5 @@ dependencies:
 - aiohttp
 # tests
 - pytest
+- pip:
+  - clisops @ git+https://github.com/roocs/clisops.git@master#egg=clisops
```

Comment:
I would skip the "pip from git" here ... I would keep it only in `requirements.txt`, as it is already.